airflow/www/views.py

```diff
@@ -1537,11 +1537,12 @@ def trigger(self, session=None):
                     'airflow/trigger.html', dag_id=dag_id, origin=origin, conf=request_conf, form=form
                 )
 
-        dag = current_app.dag_bag.get_dag(dag_id)
+        dag: DAG = current_app.dag_bag.get_dag(dag_id)
         dag.create_dagrun(
             run_type=DagRunType.MANUAL,
             execution_date=execution_date,
-            state=State.QUEUED,
+            data_interval=dag.timetable.infer_data_interval(execution_date),
+            state=State.RUNNING,
```

Comment:
```suggestion
            state=State.QUEUED,
```

Bad conflict resolve?

Reply:
Indeed.